TACC-Cloud/geoapi
# task/WG-262: Adding jwt verification for v3

## Overview: ##
Updating JWT verification function/process to match new v3 designsafe tenant 
## Related Jira tickets: ##

* [WG-262](https://tacc-main.atlassian.net/browse/WG-262)

## Summary of Changes: ##

- Updated public key to match the v3 designsafe tenant
- Added celerybeat-schedule files to .gitignore file (for cleanup)
- New claims (tapis/username and tapis/tenant_id) associated with the new signed JWT were added via WG-227

## Testing Steps: ##
1. Using this geoapi branch run 'make build'
2. Run 'make start' for backend
3. Run the angular frontend with the 'feature/tapisv3' branch (make sure your env type is set to Local in the environment.ts file in the angular/src/environments/ dir)
4. Go to localhost:4200
5. Log in and connect to the hazmapper.localhost2 client
6. Make sure you can see your project listings (local) and check the geoapi logs for any errors 

## Notes: ##
